aspnetcore/getting-started.md

```diff
@@ -23,6 +23,8 @@ uid: getting-started
     cd aspnetcoreapp
     dotnet new web
     ```
+    
+    Note: If you get an error running `dotnet new web` or you must run  `dotnet new -t web`, you'll need to upgrade to the current version of .NET Core. Install [.NET Core](https://microsoft.com/net/core)
```

Comment:
They do have to upgrade in order to follow the tutorial instructions.  This is a getting started tutorial, we should guide them to get latest if they don't have it, and I think the "Previous versions ..." text is too short, might be confusing.